docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -1,36 +1,40 @@
-# Developing a desktop viewer
+# Developing a iTwin Desktop Viewer
 
 ## Setup
 
-- [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
+- [Install necessary prerequisites]($docs/learning/tutorials/development-prerequisites).
+- From a terminal, `npx create-react-app your-app-name --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`
+  - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
+- Open the `your-app-name` directory in VS Code
+- Follow these steps to obtain a new OIDC client to use in your viewer application:
+  1. Go to <https://developer.bentley.com>
+  2. Click the **Sign In** button and sign-in using your Bentley account credentials
+      - If you have not already registered, click **Register now** and complete the registration process.
+  3. Click on your user icon and navigate to the **My Apps** page

```

Comment:
nit: there should be a "my apps" tab in the header, or can just link them to https://developer.bentley.com/my-apps/

Reply:
Good idea, I'll do that in both the Web and Desktop case.

Reply:
pushed this change.